src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -341,6 +347,21 @@ def export_model(self, model_path, mask_path=None, onnx_path=None, input_shape=N
 
         self._wrap_model()
 
+    def load_model_state_dict(self, model_state):
+        """
+        Load the state dict saved from unwrapped model.
+
+        Parameters:
+        -----------
+        model_state : dict
+            state dict saved from unwrapped model
+        """
+        if self.is_wrapped:
+            self._unwrap_model()
+            self.bound_model.load_state_dict(model_state)
+            self._wrap_model()
+        else:
```

Comment:
when will `is_wrapped`  be false ? Before calling `compress` method?

Reply:
yes.
it is possible that users call `load_model_state` before `compress`, we need an approach to check whether the model has been wrapped or not. any suggestion?